docs/docs-intro.md

```diff
@@ -10,7 +10,7 @@ sidebar_label: CITAHub 文档概览
 
 ## CITA | 区块链内核
 
-[![](https://img.shields.io/badge/CITA-Documents-green.svg)](https://cryptape.github.io/cita/)
+[![](https://img.shields.io/badge/CITA-Documents-green.svg)](https://docs.citahub.com/zh-CN/cita/cita-intro)
```

Comment:
@zhouyun-zoe Perhaps it's better that CITA also uses the Chinese version doc link.

Reply:
Ideally when all are updated we have links to zh-CN docs from Chinese pages, so for us-EN from English pages.

Reply:
> Ideally when all are updated we have links to zh-CN docs from Chinese pages, so for us-EN from English pages.

Agree with that. I mean Chinese as the first choice  maybe is better for users, check this place of CITA: 
![screenshot from 2019-02-28 13-40-26](https://user-images.githubusercontent.com/8768261/53543676-7193ff80-3b5e-11e9-893b-1527526fab77.png)

While it's English pages, so i have some doubts.

Reply:
Indeed. Need more polish work to make this better.